### PR TITLE
Past, Present, Future of Cloud Engineering - sponsors added

### PR DESCRIPTION
**Pull request by spara**
Updated Joe's post with the sponsors.

### CODE REVIEW

**Review by isaac-pulumi — COMMENTED** [review]
The added sponsor list looks good / links checked out.

File: content/blog/future-of-cloud-engineering/index.md [hunk 1]
```diff
@@ -96,3 +96,46 @@ Despite the clear gaps, it is clear we've taken some major steps in the directio
...
+                </a>
+            </div>
+            <div style="padding-top:105px;">
+                <a href="https://ns1.com/">
+                    <img src="ns1.png" alt="img" style="width:122px;height:60px;"/>
+                </a>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (cnunciato) [inline content/blog/future-of-cloud-engineering/index.md:141]: If you just want to center these images and have them wrap (and be stacked on mobile), you could simplify this with a centered flex layout, using a few [Tailwind](https://tailwindcss.com/docs/flex-direction) classes:

```suggestion
<div class="flex flex-col md:flex-row md:flex-wrap items-center justify-center">
    <a href="https://about.gitlab.com/"><img src="gitlab.png" alt="GitLab logo" class="h-16 md:mx-4 my-4"></a>
    <a href="https://www.jetstack.io/"><img src="jetstack.png" alt="Jetstack logo" class="h-16 md:mx-4 my-4"></a>
    <a href="https://aws.amazon.com/"><img src="aws.png" alt="AWS logo" class="h-16 md:mx-4 my-4"></a>
    <a href="https://jfrog.com/"><img src="jfrog.png" alt="JFrog logo" class="h-16 md:mx-4 my-4"></a>
    <a href="https://chef.io"><img src="chef.png" alt="Chef logo" class="h-16 md:mx-4 my-4"></a>
    <a href="https://ns1.com/"><img src="ns1.png" alt="NS1 logo" class="h-16 md:mx-4 my-4"></a>
</div>
```

Comment (spara) [reply content/blog/future-of-cloud-engineering/index.md:141]: Thanks, this is helpful. The tailwind examples weren't working for me, so I did the inline CSS thing. I'll try this next time.

**Review by zchase — APPROVED** [review]
LGTM

**Pull request merged**